C_code/ld_vcf.c

```diff
@@ -574,25 +568,44 @@ int main(int argc, char *argv[]) {
         abort ();
     }
   }
+  char *files[numregions];
```

Comment:
I don't know how off the top of my head but there must be a neater way to do this without manually specifying num_regions as a command line option.

You can see there's code elsewhere in this file for dynamically expanding the memory used by an array.

Having said that, it works, and I don't expect many people to be using this executable outside our API calls.

Reply:
I couldn't get dynamic arrays to work. I'll get back to it. For now I added a default value for number of allowed regions. In case someone should run the script outside of the module. I got an 'argument list too long' error when allowing 1000 regions. I set the value to 100 which works. If it becomes a common use case to pass a long list of variant IDs I'll spend more time on implementing a better solution.

Reply:
That seems reasonable